### PR TITLE
ref #248: fix height of layover container with iframes that have no i…

### PR DESCRIPTION
**Pull request by pixeljunkie**
…nitial height

| Q             | A
| ------------- | ---
| Branch        |6.2.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#248
| License       | MIT

### CODE REVIEW

File: src/MediaManagerBundle/Resources/public/js/mediaManager.js [hunk 1]
```diff
@@ -1014,7 +1014,13 @@
             var layover = self.openLayover(title, '');
 
             if (typeof height === 'undefined') {
-                height = layover.height() - layover.find('.title').height() - 50;
+
+                var height = layover.height();
```
Comment (UlrichKu) [inline src/MediaManagerBundle/Resources/public/js/mediaManager.js]: This should be sufficient: `height = $('#edit-container').height() - (layover.find('.title').height() - 50);`

The layover (opened with no html) will never have a useful height here.

File: src/MediaManagerBundle/Resources/public/css/mediaManager.css [hunk 2]
```diff
@@ -106,7 +106,7 @@
 .snippetMediaManagerBackendModuleStandard .edit-container {
     position: relative;
     padding-left: 10px;
-    min-height: 250px;
+    min-height: 450px;
```
Comment (UlrichKu) [inline src/MediaManagerBundle/Resources/public/css/mediaManager.css]: With 500px also the upload bar that appears later on will fit.

File: src/MediaManagerBundle/Resources/public/js/mediaManager.js [hunk 3]
```diff
@@ -1014,7 +1014,7 @@
             var layover = self.openLayover(title, '');
 
             if (typeof height === 'undefined') {
-                height = layover.height() - layover.find('.title').height() - 50;
+                height = $('#edit-container').height() - (layover.find('.title').height() - 50);
```
Comment (UlrichKu) [inline src/MediaManagerBundle/Resources/public/js/mediaManager.js]: This ` - (layover.find('.title').height() - 50)` is now actually a + 24px.

File: src/MediaManagerBundle/Resources/public/js/mediaManager.js (hunk 1 above)
Comment (pixeljunkie) [reply src/MediaManagerBundle/Resources/public/js/mediaManager.js]: You are right. simplified the code.

File: src/MediaManagerBundle/Resources/public/css/mediaManager.css (hunk 2 above)
Comment (pixeljunkie) [reply src/MediaManagerBundle/Resources/public/css/mediaManager.css]: thanks for finding out. changed it.

File: src/MediaManagerBundle/Resources/public/js/mediaManager.js (hunk 3 above)
Comment (pixeljunkie) [reply src/MediaManagerBundle/Resources/public/js/mediaManager.js]: I'm not 100% sure what this calculations reason was anyway? Should we change this again?

**Pull request merged**